README.md

```diff
@@ -16,15 +37,19 @@ certificates, you can visit https://hubs.local:4000 from your browser.
 
 1. [Install Docker Compose](https://docs.docker.com/compose/install)
 2. [Install Mutagen Compose](https://github.com/mutagen-io/mutagen-compose#system-requirements)
-3. Add these entries to your hosts file:
+3. [Install Mutagen](https://mutagen.io/documentation/introduction/installation)
```

Comment:
Since we’re adding this to the README, we’ll want to include the important qualifier from https://github.com/mutagen-io/mutagen-compose#system-requirements:
> a **matching version** of Mutagen

Emphasis added.

We’ll also want to list it above Mutagen Compose since it’s a dependency thereof.